journal/modernized/32-epistles.adoc

```diff
@@ -218,17 +225,21 @@ or encumbered with the things of this life; but be ready for sufferings,
 which may attend that nation before many be aware, that we may all be ready,
 as Christ`'s freemen, to drink that cup which the Lord is pleased to put into our hands,
 for the trial of our faith, which is more precious than gold.
+
+[.signed-section-closing]
 My dear and true love is with you all, in the power of an endless life,
 wherein I am Your friend and brother,
 
-William Edmundson.
+[.signed-section-signature]
+William Edmundson
 
 === Some Letters to His Wife
```

Comment:
This one also needs to be changed to a [.letter-heading], instead of a === subheading